src/app/pages/knowledge-list-item-log/knowledge-list-item-controller.js

```diff
@@ -0,0 +1,35 @@
+angular.module('knowledgeList').
+controller('KnowledgeListItemCtrl',
+  [ '$scope',
+    '$state',
+    'usersRepository',
+    '$stateParams',
+    'authorization',
+    'config',
+    '$http',
+
+    function ($scope,
+              $state,
+              $stateParams,
+              usersRepository,
+              authorization,
+              config,
+              $http) {
+      'use strict';
+
+      var userId = $stateParams.userId,
+          knowledgeLogData;
+
+      $http.get('/api/logs')
```

Comment:
Controller should not use "$http" service directly. You should move this call to separate service. Use that service here.